External/FEXCore/Source/Interface/IR/Passes/RegisterAllocationPass.cpp

```diff
@@ -565,8 +799,21 @@ namespace FEXCore::IR {
       for (uint32_t j = i + 1; j < NodeCount; ++j) {
         if (!(LiveRanges[i].Begin >= LiveRanges[j].End ||
               LiveRanges[j].Begin >= LiveRanges[i].End)) {
-          AddInterference(i, j);
-          AddInterference(j, i);
+          
```

Comment:
Not sure if this is the way we want to do. @Sonicadvance1 wants to keep cross-class interferences and resolve them later... but where?